tools/stamp_file.bzl

```diff
@@ -1,7 +1,5 @@
-load("//cc:defs.bzl", "swift_cc_library")
-
-def _stamping_impl(ctx):
-    status_file = ctx.version_file
+def _stamp_file_impl(ctx):
+    status_file = ctx.info_file
```

Comment:
there's two files that result from `workspace_status`:
- `version_file` - contains volatile status variables
- `info_file` - contains stable status variables

We should probably handle both files.